pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -44,7 +48,10 @@ func (pl *DefaultPreemption) Name() string {
 
 // New initializes a new plugin and returns it.
 func New(_ runtime.Object, fh framework.FrameworkHandle) (framework.Plugin, error) {
-	pl := DefaultPreemption{fh}
+	pl := DefaultPreemption{fh: fh}
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.PodDisruptionBudget) {
+		pl.pdbLister = fh.SharedInformerFactory().Policy().V1beta1().PodDisruptionBudgets().Lister()
```

Comment:
@ahg-g to avoid temporary signature change, the pdbLister is defined here but not used in Preempt(). I will use the pdbLister in the next followup - where Preempt() gets moved to defaultpreemption package.